README.md

```diff
@@ -1,6 +1,10 @@
 # Northcoders News API
 
-For instructions, please head over to [L2C NC News](https://l2c.northcoders.com/courses/be/nc-news).
+For instructions, 
+--you need to install dotenv package.
+-- make two  files starts with ".env", one for development perposes one for testing . 
+--then set the environment variables  with your database name considering the testing and developing database names . 
```

Comment:
Your `setup.sql` specifies what the database names need to be, it would be helpful to specify them here. Consider writing here exactly what needs to be in these files, so that anyone who wants to run your project can do it easily